tests/src/Integration/Action/EntityPropertyFetchTest.php

```diff
@@ -0,0 +1,210 @@
+    $query->expects($this->once())
+      ->method('condition')
+      ->with($property_name, $property_value, '=')
+      ->will($this->returnValue($query));
+    $query->expects($this->once())
+      ->method('range')
+      ->with(0, $limit)
+      ->will($this->returnValue($query));
+    $query->expects($this->once())
+      ->method('execute')
+      ->will($this->returnValue($entity_ids));
+
+    // Create dummy entity storage object.
+    $entityStorage = $this->getMock('Drupal\Core\Entity\EntityStorageInterface');
+    $entityStorage->expects($this->once())
+      ->method('loadMultiple')
+      ->with($entity_ids)
+      ->will($this->returnValue($entities));
+    $entityStorage->expects($this->once())
+      ->method('getQuery')
+      ->will($this->returnValue($query));
+    $this->entityManager->expects($this->once())
+      ->method('getStorage')
+      ->with($entity_type)
+      ->will($this->returnValue($entityStorage));
+
+
+    // Set context values for EntityPropertyFetch action and execute.
+    $this->action->setContextValue('type', $entity_type)
+      ->setContextValue('property', $property_name)
+      ->setContextValue('value', $property_value)
+      ->setContextValue('limit', $limit)
+      ->execute();
+
+    // Test that executing action with a value for limit returns the dummy entities array.
+    $this->assertEquals($entities, $this->action->getProvided('entity_fetched')->getContextValue('entity_fetched'));
+  }
+
+  /**
+   * Tests that the context provided by the action execution has the correct entity type.
+   *
+   * @covers ::execute()
+   */
+  function testActionExecutionProvidedContextEntityType() {
+    // Create variables for action context values.
+    $entity_type = 'entity_test';
+    $property_name = 'test_property';
+    $property_value = 'llama';
+
+    // Create an array of dummy entities.
+    $entities = [];
+    for ($i = 0; $i < 2; $i++) {
+      $entity = $this->getMock('Drupal\Core\Entity\EntityInterface');
+      $entities[] = $entity;
+    }
+
+    // Create dummy entity storage object.
+    $entityStorage = $this->getMock('Drupal\Core\Entity\EntityStorageInterface');
+    $entityStorage->expects($this->once())
+        ->method('loadByProperties')
+        ->with(array($property_name => $property_value))
+        ->will($this->returnValue($entities));
+    $this->entityManager->expects($this->once())
+        ->method('getStorage')
+        ->with($entity_type)
+        ->will($this->returnValue($entityStorage));
+
+    // Set context values for EntityPropertyFetch action and execute.
+    $this->action->setContextValue('type', $entity_type)
+        ->setContextValue('property', $property_name)
+        ->setContextValue('value', $property_value)
+        ->execute();
+
+    // Test that the provided context has the correct entity type.
+    $this->assertEquals('entity:' . $entity_type, $this->action->getProvidedDefinition('entity_fetched')->getDataType());
+  }
+
+}
```

Comment:
I guess there could be an issue with travis configuration. I don't know what is in composer.json and why it wants to patch drupal core. Try to ask at d.o. issue